modules/vaos/spec/requests/vaos/v2/appointments_spec.rb

```diff
@@ -913,12 +923,34 @@ def stub_clinics
                        'd12672eba61b7e9bc50bb6085a0697133a5fbadf195e6cade452ddaad7921c1d/appointments/00000'
             get '/vaos/v2/appointments/00000'
             body = JSON.parse(response.body)
+
             expect(response).to have_http_status(:bad_gateway)
             expect(body.dig('errors', 0, 'code')).to eq('VAOS_502')
             expect(body.dig('errors', 0, 'source', 'vamf_url')).to eq(vamf_url)
           end
         end
       end
+
+      context 'when the EPS service errors on retrieving an appointment' do
+        before do
+          allow(Flipper).to receive(:enabled?).with(:va_online_scheduling_use_vpg, instance_of(User)).and_return(false)
+          allow(Flipper).to receive(:enabled?).with(:va_online_scheduling_vaos_alternate_route).and_return(false)
+        end
+
+        it 'returns a 502 status code' do
+          VCR.use_cassette('vaos/eps/token/token_200', match_requests_on: %i[method path query]) do
+            VCR.use_cassette('vaos/eps/get_appointment/500', match_requests_on: %i[method path query]) do
+              vamf_url = 'https://api.wellhive.com/care-navigation/v1/appointments/qdm61cJ5?retrieveLatestDetails=true'
+              get '/vaos/v2/appointments/qdm61cJ5?_include=eps', headers: inflection_header
+              expect(response).to have_http_status(:bad_gateway)
+              body = JSON.parse(response.body)
+
+              expect(body.dig('errors', 0, 'code')).to eq('VAOS_502')
+              expect(body.dig('errors', 0, 'source', 'vamfUrl')).to eq(vamf_url)
```

Comment:
As per my note at the bottom of the pr writeup in the requested feedback section, the key for the `vamfUrl` is in camel case, in the test just above this for VAOS upstream error the key is in snake case.

Reply:
This is something we can debate after the fact, but consistency would be nice. I am sure APIs in the repo do all kinds of things.